provider/data/experimental/data/percent_essentials_v1_marker.rs.data

```diff
@@ -4,8 +4,8 @@
 /// `icu`'s `_unstable` constructors.
 ///
 /// Using this implementation will embed the following data in the binary's data segment:
-/// * 267B for the lookup data structure (59 data identifiers)
-/// * 1035B[^1] for the actual data (9 unique structs)
+/// * 223B for the lookup data structure (47 data identifiers)
+/// * 196B[^1] for the actual data (7 unique structs)
```

Comment:
Kinda cool to see just _how much_ smaller that data is when using `icu_pattern`!